app/views/users/sessions/new.html.erb

```diff
@@ -2,29 +2,25 @@
   <h2>Log in</h2>
```

Comment:
Given that we're merging the sign-in/sign-up process, I feel like this button should reflect that too - to me, "Log in" is a button I'd assume requires me to already have an account.

In the same vein, we should probably standardise our use of "Log in/out" vs "Sign in/out". I don't mind which one we use, but it clashes a little that you press the `Log in` button to reach the page, but once entering your details the button at the bottom reads `Sign in/up`.

Reply:
That's a good point. We could add a 'Sign up' option to the nav bar that navigates directly to the sign up form.

Idea for changing the UX:
Log in button and sign up buttons on nav bar
Log in form has 'Log in' title and 'log in' button. But, if the username isn't recognised the log in button navigates to the sign up form, auto populates the relevant fields and gives a message saying the username wasn't recognised.

This could potentially be annoying if the user just misspells their username or something but maybe that's not too important

That should remove the clash of sign in / log in. 

Alternatively we could just use the term 'Sign in' everywhere rather than 'log in' and not change the UX. Could do this for now and return to faff more with it later

Reply:
This feels like the kind of thing to get input from @ColonelPanicks on. Having a single page where you enter your info which can either log you in or sign you up (depending on if you exist) sounds great, but as you say it would be pretty annoying if you misspelled your username and were brought to a separate signup screen instead of just getting a little popup saying "your details were incorrect" or something similar.